frontend/nuxt.config.ts

```diff
@@ -172,7 +159,6 @@ export default defineNuxtConfig({
     cssPath: "~/styles/tailwind.css",
   },
   i18n: {
```

Comment:
It looks like i18n can be configured with runtime config as well, and potentially a mix, but might require code changes to do so: https://i18n.nuxtjs.org/docs/options/runtime-config